cmd/loadtest/submitter/submitter.go

```diff
@@ -120,7 +126,7 @@ func (s *Submitter) StartSubmission(ctx context.Context, limiter *rate.Limiter,
 // The transaction has the same property on every call:
 // Same source and and desitnation addresses, and same amount.
 // The only property that changes is the sequence number.
-func (s *Submitter) submit(logger log.Logger, destIndex int, native bool) error {
+func (s *Submitter) submit(logger log.Logger, destIndex int, native bool, clientIndex int) error {
```

Comment:
why not just transfer a reference to the client object i.e. `*horizon.Client` instead of its index?

Reply:
Because I don't know go, so I hack to make it work.